content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+configuração no kubelet. Os limites de PID definidos pelo pod não são suportados no momento.
+
+{{< caution >}}
+Isso significa que o limite aplicável a um pod pode ser diferente dependendo
+onde o pod está alocado. Para tornar as coisas simples, é mais fácil se todos os nós usarem
+os mesmos limites e reservas de recursos PID.
+{{< /caution >}}
+
+## Limites de PID de nós
+
+O Kubernetes permite que você reserve vários IDs de processo para uso do sistema. Para
+configurar a reserva, use o parâmetro `pid=<número>` no
+opções de linha de comando `--system-reserved` e `--kube-reserved` para o kubelet.
+O valor especificado declara que o número especificado de IDs de processo será
+reservado para o sistema como um todo e para os daemons do sistema Kubernetes
+respectivamente.
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, limitação de recursos PID com nível de nós
+reservas necessárias habilitando o [`feature-gates`
+](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportNodePidsLimit` para funcionar.
+{{< /note >}}
+
+## Limites de PID do pod
+
+O Kubernetes permite limitar o número de processos em execução em um pod. Você
+especifica esse limite no nível do nó, em vez de configurá-lo como um recurso
+limite para um pod específico. Cada nó pode ter um limite de PID diferente.
+Para configurar o limite, você pode especificar o parâmetro de linha de comando `--pod-max-pids`
+para o kubelet ou definir `PodPidsLimit` no arquivo de configuração do kubelet
+[arquivo de configuração](/docs/tasks/administer-cluster/kubelet-config-file/).
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, era necessário habilitar a limitação de recursos PID para pods
+o [portão de recursos](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportPodPidsLimit` para funcionar.
+{{< /note >}}
+
+## Remoção baseada em PID
+
+Você pode configurar o kubelet para iniciar o encerramento de um pod quando ele estiver se comportando mal e consumindo uma quantidade anormal de recursos.
+Esse recurso é chamado de despejo. Você pode
+[Configurar fora da manipulação de recursos](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
+para vários sinais de despejo.
+Use o sinal de remoção `pid.available` para configurar o limite para o número de PIDs usados pelo pod.
+Você pode definir políticas de despejo flexíveis e rígidos.
+Porém, mesmo com a política de despejo rígido, se o número de PIDs crescer muito rápido,
+o nó ainda pode entrar em estado instável ao atingir o limite de PIDs do nó.
+O valor do sinal de despejo é calculado periodicamente e NÃO impõe o limite.
+
+Limitação de PID - por pod e por nó define o limite rígido.
+Assim que o limite for atingido, a carga de trabalho começará a apresentar falhas ao tentar obter um novo PID.
+Pode ou não levar ao reagendamento de um Pod,
+dependendo de como a carga de trabalho reage a essas falhas e como vivacidade e prontidão
```

Comment:
suggestion: "...e como as sondas de saúde (liveness e readiness) do pod estão configuradas."

Reply:
@devlware do you feel strongly about using "saúde"? I think in other pages we have used "integridade" instead.

Reply:
integridade is much better @stormqueen1990!